templates/CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+5. Make your changes.
+6. [Submit the branch as a PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) pointing to the `master` branch of the main XXX repo. A maintainer should comment and/or review your Pull Request within a few days. Although depending on the circumstances, it may take longer.<br>
+ We do not enforce a naming convention for the PRs, but **please use something descriptive of your changes**, having in mind that the title of your PR will be automatically added to the next [release changlogs](https://github.com/meilisearch/XXX/releases/).
+
+## Development Workflow
+
+---
+
+COMMENT TO REMOVE
+
+This section should be filled according to the repository and should contain:
+- A "Setup" section (if needed) with the command to install the dependencies for the contributor (e.g.: `yarn --dev`).
+- A "Tests and Linter" section with the command to run the tests, the linter and the linter-autofix (if possible).
+- A "Release Process" section that describes the process before publishing a new release.
+- Other useful tips/steps for the contributor (e.g.: how to build, how to debug).
+
+/!\ The following lines are just a template example and should be completed with right names and right commands
+
+---
+
+### Setup
+
+```bash
+$ <Add the right command here>
+```
+
+### Tests and Linter
+
+Each PR should pass the tests and the linter to be accepted.
+
+```bash
+# Tests
+$ <Add Docker command to run MeiliSearch here>
+$ <Add the right command here>
+# Linter
+$ <Add the right command here>
+# Linter with fixing
+$ <Add the right command here>
+```
+
+### Release Process
+
+MeiliSearch tools follow the [Semantic Versioning Convention](https://semver.org/).
+
+#### Automated Changelogs
+
+For each PR merged on `master`, a GitHub Action is running and updates the next release description as a draft release in the [GitHub interface](https://github.com/meilisearch/XXX/releases). If you don't have the right access to this repository, you will not be able to see the draft release until the release is published.
+
+The draft release description is therefore generated and corresponds to all the PRs titles since the previous release. This means each PR should only do one change and the title should be desriptive of this change.
+
+About this automation:
+- As the draft release description is generated on every push on `master`, don't change it manually until the final release publishment.
+- If you don't want a PR to appear in the release changelogs: set the label `skip-changelog`. We used to remove PRs updating the README or the CI (except for big changes).
+- If the changes you are doing in the PR are breaking: set the label `breaking-change`. In the release tag, the minor will be increased instead of the patch. The major will never be changed until [MeiliSearch](https://github.com/meilisearch/MeiliSearch) is stable.
```

Comment:
```suggestion
- If the changes you are doing in the PR are breaking: add the label `breaking-change`. In the release tag, the minor will be increased instead of the patch. The major will never be changed until [MeiliSearch](https://github.com/meilisearch/MeiliSearch) is stable.
```